pwiz_tools/Skyline/TestPerf/SmallMolLibrariesTutorialTest.cs

```diff
@@ -0,0 +1,431 @@
+        [Timeout(int.MaxValue)] // These can take a long time
+        public void TestSmallMoleculeLibrariesTutorial()
+        {
+//            IsPauseForScreenShots = true;
+//            IsPauseForCoverShot = true;
+            CoverShotName = "SmallMolLibraries";
+
+            LinkPdf = "https://skyline.ms/_webdav/home/software/Skyline/%40files/tutorials/SmallMoleculeIMSLibraries.pdf";
+
+            TestFilesZipPaths = new[]
+            {
+                @"https://skyline.ms/tutorials/SmallMoleculeLibraries.zip",
+                @"TestPerf\SmallMolLibrariesTutorialViews.zip",
+            };
+
+            TestFilesPersistent = new[] { Flies_M, Flies_F };
+
+            RunFunctionalTest();            
+        }
+
+        private string DataPath { get { return TestFilesDirs.Last().PersistentFilesDir; } }
+
+        protected override void DoTest()
+        {
+            // Setting the UI mode, p 2  
+            var startPage = WaitForOpenForm<StartPage>();
+            RunUI(() => startPage.SetUIMode(SrmDocument.DOCUMENT_TYPE.proteomic));
+            PauseForScreenShot<StartPage>("Start Window proteomic", 2);
+            RunUI(() => startPage.SetUIMode(SrmDocument.DOCUMENT_TYPE.small_molecules));
+            PauseForScreenShot<StartPage>("Start Window small molecule", 3);
+            ShowSkyline(() => startPage.DoAction(skylineWindow => true));
+            PauseForScreenShot("Empty Skyline window", 3);
+
+            //   �	On the Settings menu, click Default.
+            //   �	Click No on the form asking if you want to save the current settings.
+            RunUI(() => SkylineWindow.ResetDefaultSettings());
+            MaximizeSkylineWindow(); // Set the Skyline window large enough to accomodate all windows for screenshots
+
+            var doc = SkylineWindow.Document;
+
+            //   �	On the Settings menu, click Transition Settings.
+            //   �	Click the Filter tab.
+            var transitionSettingsUI = ShowDialog<TransitionSettingsUI>(() =>
+                SkylineWindow.ShowTransitionSettingsUI(TransitionSettingsUI.TABS.Filter));
+
+            //   �	This data was collected in negative ionization mode so [M+H] and[M +] can be removed from the Precursor Adducts and Fragment Adducts fields. However, they are harmless if left as is since the library we will use has only negative ion mode entries.
+            //   �	In the Precursor Adducts field, enter �[M-H], [M+HCOO], [M+CH3COO]�. 
+            //   �	In the Fragment Adducts field, enter �[M-]�.
+            RunUI(() =>
+            {
+                transitionSettingsUI.SmallMoleculePrecursorAdducts = "[M-H], [M+HCOO], [M+CH3COO]";
+                transitionSettingsUI.SmallMoleculeFragmentAdducts = "[M-]";
+                transitionSettingsUI.SmallMoleculeFragmentTypes = "f, p";
+            });
+            //   �	The Transition Settings form should now look like this:
+            PauseForScreenShot("Transition Settings: Filter", 4);
```

Comment:
Use the IForView type in this pause for the Transition Settings - Filter tab

Reply:
right, thanks.